docs/pages/guides/customizing-metro.md

```diff
@@ -19,7 +26,90 @@ const config = getDefaultConfig(__dirname);
 module.exports = config;
 ```
 
-You can find a complete list of supported options in the Metro docs: [Configuring Metro](https://facebook.github.io/metro/docs/configuration). Please note that you only need to specify the options that you want to customize: the custom config will be merged with the defaults from `expo/metro-config` when using [Expo CLI][expo-cli] to start dev servers (`npx expo start`) or export static files (`npx expo export`).
+To learn more, refer to the [`metro.config.js` docs](https://facebook.github.io/metro/docs/configuration).
+
+## Assets
+
+Metro resolves files as either source code or assets. Source code is JavaScript, TypeScript, JSON, and other files that are used by your application. [Asset](/guides/assets) are images, fonts, and other files that should not be transformed by Metro. To accommodate large-scale codebases, Metro requires all extensions for both source code and assets to be explicitly defined before starting the bundler. This is done by adding the `resolver.sourceExts` and `resolver.assetExts` options to the Metro configuration. By default we include the following extensions:
+
+- [`resolver.assetExts`](https://github.com/facebook/metro/blob/main/packages/metro-config/src/defaults/defaults.js#L15)
+- [`resolver.sourceExts`](https://github.com/facebook/metro/blob/main/packages/metro-config/src/defaults/defaults.js#L15)
+
+### Adding more file extensions to `assetExts`
+
+The most common customization is including extra asset extensions to Metro. In the **metro.config.js** file, add the file extension (without a leading `.`) to `resolver.assetExts`:
+
+```js
+// metro.config.js
+const { getDefaultConfig } = require('@expo/metro-config');
+
+const config = getDefaultConfig(__dirname);
+
+config.resolver.assetExts.push(
+  // Adds support for `.db` files for SQLite databases
+  'db'
+);
+
+module.exports = config;
+```
+
+## Minification
+
+Source code is optimized for readability and debugging. To optimize your application for performance, the source code is automatically minified when [compiling](/workflow/expo-cli#compiling) and [exporting](/workflow/expo-cli#exporting). You can also minify your code during development with `npx expo start --minify`, this is sometimes useful for testing production optimizations.
+
+By default, Metro uses `uglify-es` to minify code. According to [this benchmark](https://github.com/privatenumber/minification-benchmarks) `uglify` generally produces the smallest bundles, and is nearly the _slowest_ minifier. Here are some alternative minifiers you can use with Metro.
+
+### Minification: esbuild
+
+You can use [`esbuild`](https://esbuild.github.io/) to minify exponentially faster than `uglify-es` and `terser`. Just follow this guide: [`metro-minify-esbuild` usage](https://github.com/EvanBacon/metro-minify-esbuild#usage).
+
+### Minification: Terser
+
+You can use [`terser`](https://github.com/terser/terser) instead of `uglify-es` to mangle and compress your project.
```

Comment:
we might want to put uglify first if we are explaining terser in reference to uglify

Reply:
We explain the uglify part in the main section under "Minification"